Revit_Core_Engine/Query/BHoMType.cs

```diff
@@ -636,8 +618,22 @@ public static Type BHoMType(this CurveElement curveElement, Discipline disciplin
                 default:
                     return typeof(BH.oM.Adapters.Revit.Elements.IInstance);
             }
+        }
 
-            return null;
+        /***************************************************/
+
+        [Description("Finds a suitable BHoM type to convert the given Revit AssemblyInstance to, based on the requested engineering discipline and adapter settings.")]
+        [Input("assemblyInstance", "Revit AssemblyInstance to find a correspondent BHoM type.")]
+        [Input("discipline", "Engineering discipline based on the BHoM discipline classification.")]
+        [Input("settings", "Revit adapter settings to be used while performing the search for the correspondent type.")]
+        [Output("bHoMType", "A suitable BHoM type to convert the given Revit AssemblyInstance to.")]
+        public static Type BHoMType(this AssemblyInstance assemblyInstance, Discipline discipline, RevitSettings settings = null)
+        {
+            switch (discipline)
+            {
+                default:
+                    return typeof(BH.oM.Adapters.Revit.Elements.Assembly);
+            }
```

Comment:
I wanted to stick to the convention used across the entire file, to make sure the code is readable and easy to extend if needed. If we were to change it, I would align all such cases in the entire file in a separate PR. However, personally I like the current setup because all methods have the exact same structure.